test-clients/src/main/java/com/hedera/services/bdd/suites/token/UniqueTokenManagementSpecs.java

```diff
@@ -106,10 +112,56 @@ public static void main(String... args) {
 						validatesQueryOutOfRange(),
 						failsWithInvalidQueryBoundaries(),
 						getAccountNftsInfoFailsWithDeletedAccount(),
-						getAccountNftsInfoFailsWithInexistentAccount()
+						getAccountNftsInfoFailsWithInexistentAccount(),
+						tokenDissociateHappyPath(),
+						tokenDissociateFailsIfAccountOwnsUniqueTokens()
                 }
         );
     }
+	private HapiApiSpec tokenDissociateFailsIfAccountOwnsUniqueTokens() {
+		return defaultHapiSpec("tokenDissociateFailsIfAccountOwnsUniqueTokens")
+				.given(
+						newKeyNamed(SUPPLY_KEY),
+						cryptoCreate(TOKEN_TREASURY),
+						cryptoCreate("acc"),
+						tokenCreate(NFT)
+								.initialSupply(0)
+								.tokenType(TokenType.NON_FUNGIBLE_UNIQUE)
+								.supplyType(TokenSupplyType.INFINITE)
+								.supplyKey(SUPPLY_KEY)
+								.treasury(TOKEN_TREASURY)
+				).when(
+						tokenAssociate("acc", NFT),
+						mintToken(NFT, List.of(metadata("memo1"), metadata("memo2")))
+				).then(
+						cryptoTransfer(TokenMovement.movingUnique(1L, NFT).between(TOKEN_TREASURY, "acc")),
+						cryptoTransfer(TokenMovement.movingUnique(2L, NFT).between(TOKEN_TREASURY, "acc")),
+						tokenDissociate("acc", NFT).hasKnownStatus(FAIL_INVALID)
+				);
+	}
+
+	private HapiApiSpec tokenDissociateHappyPath(){
+		return defaultHapiSpec("tokenDissociateHappyPath")
+				.given(
+
+						newKeyNamed(SUPPLY_KEY),
+						cryptoCreate(TOKEN_TREASURY),
+						cryptoCreate("acc"),
+						tokenCreate(NFT)
+								.initialSupply(0)
+								.tokenType(TokenType.NON_FUNGIBLE_UNIQUE)
+								.supplyType(TokenSupplyType.INFINITE)
+								.supplyKey(SUPPLY_KEY)
+								.treasury(TOKEN_TREASURY)
+				)
+				.when(
+						tokenAssociate("acc", NFT),
+						mintToken(NFT, List.of(metadata("memo1"), metadata("memo2")))
+				)
+				.then(
+						tokenDissociate("acc", NFT).hasKnownStatus(SUCCESS)
```

Comment:
We can assert the token relationships for the given account to verify the "effect" of the operation